build-system/common/ci.js

```diff
@@ -170,13 +170,8 @@ function ciJobId() {
  * Returns the URL of the current job.
  * @return {string}
  */
-function ciJobUrl() {
-  return isGithubActions
```

Comment:
Don't we still use GitHub Actions? https://github.com/ampproject/amphtml/actions/workflows/cross-platform-builds.yml

Reply:
We do, but we don't use this function anywhere in GHA - figured I'd remove it as a dangling TODO rather than try to fix the TODO for no reason